core/src/main/java/io/opencensus/stats/Aggregation.java

```diff
@@ -194,54 +122,52 @@ static Range create() {
      *
      * @return a new {@code Mean}.
      */
-    // TODO(songya): not v0.1, expose factory method later.
-    static Mean create() {
+    public static Mean create() {
```

Comment:
I think Mean (and Sum and Count) are parameterless so you can create a static constant that you always return.

Reply:
Sure, updated.